digdag-cli/src/main/java/io/digdag/cli/client/ClientCommand.java

```diff
@@ -102,6 +106,10 @@ protected DigdagClient buildClient(boolean checkServerVersion)
             endpoint = props.getProperty("client.http.endpoint", DEFAULT_ENDPOINT);
         }
 
+        if ( disableDirectDownload == null) { //CLI option should have priority to system config
```

Comment:
style problem:
```suggestion
        if (disableDirectDownload == null) {  // CLI option should have priority to system config
```

Reply:
Thanks, will fix it.

Reply:
I removed this line because CLI option has been removed.